ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala

```diff
@@ -96,13 +96,15 @@ object TrustManagerConfig {
  *
  * A trust store must either provide a file path, or a data String.
  *
- * @param storeType The store type. Defaults to the platform default store type (ie, JKS).
- * @param filePath The path of the key store file.
  * @param data The data to load the key store file from.
+ * @param filePath The path of the key store file.
+ * @param password The password to use to load the key store file, if the file is password protected.
+ * @param storeType The store type. Defaults to the platform default store type (ie, JKS).
  */
 final class TrustStoreConfig private[sslconfig](
   val data: Option[String],
   val filePath: Option[String],
+  val password: Option[String] = None,
```

Comment:
Let's avoid default parameters as much as we can btw. They're hell for binary compatibility